Makefile

```diff
@@ -79,6 +81,11 @@ SWAGGER_DOCS_PORT ?= 9000
 INTEGRATION_CLI_MASTER_IMAGE := $(if $(INTEGRATION_CLI_MASTER_IMAGE), $(INTEGRATION_CLI_MASTER_IMAGE), integration-cli-master)
 INTEGRATION_CLI_WORKER_IMAGE := $(if $(INTEGRATION_CLI_WORKER_IMAGE), $(INTEGRATION_CLI_WORKER_IMAGE), integration-cli-worker)
 
+define \n
```

Comment:
I tried that the other day and couldn't get it to work, I guess my Makefile was subawsome...